niaaml/classifiers/__init__.py

```diff
@@ -23,9 +28,14 @@
     "Bagging",
     "ExtremelyRandomizedTrees",
     "DecisionTree",
+    "DecisionTreeRegression",
```

Comment:
I opted for the post-fix "Regression" to indicate if a classifier is a regressor.

Reply:
Perfect.